src/components/MultiBarToggle.js

```diff
@@ -134,6 +137,14 @@ class MultiBarToggle extends Component {
                     >
                         {route.icon}
                     </AnimatedTouchable>
+                    {route.buttonlabel &&
+                        <Text style={[Styles.actionContent, {
+                            color: 'white',
+                            // width: actionSize,
+                            fontSize: 9,
```

Comment:
Move styles properties to the styles.

Reply:
actionContentText: {
        flex: 1,
        alignItems: 'center',
        justifyContent: 'center',
        color: 'white',
        fontSize: 9,
        textAlign: 'center',
        paddingTop: 5
    },